reports/pharmacy_first_report.Rmd

```diff
@@ -67,6 +67,34 @@ df_measures <- tidy_measures(
   pf_measures_name_mapping = pf_measures_name_mapping,
   pf_measures_groupby_dict = pf_measures_groupby_dict
 )
+
+df_measures$ethnicity <- factor(
+  df_measures$ethnicity,
+  levels = c("White", "Mixed", "Asian or Asian British",
+  "Black or Black British", "Chinese or Other Ethnic Groups",
+  "Missing"), 
+  ordered = TRUE
+)
+
+df_measures$age_band <- factor(
+  df_measures$age_band,
+  levels = c("0-19", "20-39", "40-59",
+  "60-79", "80+",
+  "Missing"), 
+  ordered = TRUE
+)
+
+df_measures$region <- factor(
+  df_measures$region,
+  levels = c("East", "East Midlands", "London",
+  "North East", "North West", "South East",
+  "South West", "West Midlands", "Yorkshire and The Humber",
+  "Missing"), 
+  ordered = TRUE
+)
```

Comment:
this does what we want, but it would be could do this as one call using mutate. at some later point we should integrate this into the `tidy_measures()` function, for now I think it's ok if we do this outside of the function.

```R
df_measures <- df_measures %>%
  mutate(
    ethnicity = factor(ethnicity, ...),
    region = factor(region, ...),
    age_band = factor(age_band, ...),
    sex = factor(sex, ...)
)
```